src/renderer/views/wallet/UnlockView.tsx

```diff
@@ -1,25 +1,15 @@
-import React, { useEffect } from 'react'
+import React from 'react'
 
 import { useObservableState } from 'observable-hooks'
-import { useHistory } from 'react-router'
 
 import UnlockForm from '../../components/wallet/UnlockForm'
 import { useWalletContext } from '../../contexts/WalletContext'
-import * as appRoutes from '../../routes/app'
 import { initialKeystoreState } from '../../services/wallet/service'
-import { hasImportedKeystore } from '../../services/wallet/util'
 
 const UnlockView: React.FC = (): JSX.Element => {
   const { keystoreService } = useWalletContext()
   const { keystore$, removeKeystore } = keystoreService
   const keystore = useObservableState(keystore$, initialKeystoreState())
-  const history = useHistory()
-
-  useEffect(() => {
```

Comment:
why did you decide to remove this ?

Reply:
Route update for unlocking the wallet does work without it. I've found it out by an accident after disabling these lines for debugging.